qiskit/transpiler/passes/routing/sabre_swap.py

```diff
@@ -451,8 +451,8 @@ def _shortest_swap_path(target_qubits, coupling_map, layout):
     virtual qubits in ``target_qubits`` together in the coupling map."""
     v_start, v_goal = target_qubits
     start, goal = layout._v2p[v_start], layout._v2p[v_goal]
-    # TODO: remove the list call once using retworkx 0.12, as the return value can be sliced.
-    path = list(retworkx.dijkstra_shortest_paths(coupling_map.graph, start, target=goal)[goal])
+    # TODO: remove the list call once using reseaux 0.12, as the return value can be sliced.
+    path = list(reseaux.dijkstra_shortest_paths(coupling_map.graph, start, target=goal)[goal])
```

Comment:
We should change this line properly, since v0.12 will bring about the change the TODO is talking about.